content/telegraf/v1.18/about_the_project/release-notes-changelog.md

```diff
@@ -10,6 +10,29 @@ menu:
     parent: About the project
 ---
 
+## v1.18.2 [2021-04-29]
+
+- Make JSON format compatible with nulls to ensure Telegraf successfully detects null values and returns an empty metric without error.
+- Update `common.shim` by changing `NewStreamParser` to accept larger inputs from scanner.
```

Comment:
not sure whether common.shim is plugin?

Reply:
and/or whether it makes sense to fit lines 15-16 into a plugin update

Reply:
common.shim isn't a plugin, so it makes sense here.